localstack-core/localstack/services/lambda_/provider.py

```diff
@@ -2093,6 +2097,10 @@ def create_function_url_config(
                 # just for this particular lambda function, but for the entire
                 # lambda provider. Therefore... that idea proved non-trivial!
                 url_id = fn.tags[TAG_KEY_CUSTOM_URL]
+                if not TAG_KEY_CUSTOM_URL_VALIDATOR.match(url_id):
+                    raise ValidationException(
```

Comment:
nit: maybe we could add a logging statement explaining with a bit more detail why the custom tag was not accepted (what the custom tag value is, the rules and to update the tag?), so that the user can quickly see why there was an error?

Reply:
I was thinking more of a pure logging statement available in LocalStack logs, as this exception will be returned to the Client (which is good!) but it can also be good to log it directly.

As a note, we usually don't log this type of expression, because we're only aiming for parity with AWS, so those errors are expected and would also be raised by AWS.

In this case, it's a bit more subtle, which is why I was wondering about a logging statement. 

Something maybe similar to this, but I haven't put much thought into it. 
```python
LOG.warning("The value of your '%s' Tag ('%s') does not satisfy the follow constraints (describe the constraints). Please update the value of tag to be able to create your Lambda URL configuration.", TAG_KEY_CUSTOM_URL, url_id)
```